primeqa/pipelines/README.md

```diff
@@ -35,31 +40,39 @@ retriever.load()
 
 - Step 2:  Initialize the reader model. 
 
+We can use a reader based on ChatGPT.
+
 ```python
-reader = ExtractiveReader(model_name_or_path = model)
+reader = PromptGPTReader(api_key='API KEY HERE', model_name="gpt-3.5-turbo")
```

Comment:
Should we also add an example with Instruct-GPT here?

Reply:
To do instruct gpt we can just change the model name. I have it just above the code example:
```
We prompt pre-trained large language models (LLMs) to answer questions conditioned on the retrieved passages.
We support integration with the OpenAI's ChatGPT(gpt-3.5-turbo) and InstructGPT(text-davinci-003), using an OpenAI API key (https://platform.openai.com/account/api-keys).
```